documentation/book/assembly-kafka-cluster.adoc

```diff
@@ -0,0 +1,39 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {ContextProduct}
+
+include::common/attributes.adoc[]
+
+[id='kafka-broker-{context}']
+= Kafka cluster
+
+When installing Kafka, {ProductName} also installs a Zookeeper cluster and adds the necessary configuration for the Kafka broker connection.
+
+{ProductName} uses `StatefulSets` to deploy Kafka brokers.
```

Comment:
These 3 lines (17-19) should have been kicked out together with the part about services.